packages/webapp/src/components/Filter/FilterButton/styles.module.scss

```diff
@@ -0,0 +1,50 @@
+.filterButtonContainer {
+  position: relative;
+  height: 100%;
+  width: 40px;
+  display: flex;
+  justify-content: center;
+  background-color: var(--grey200);
+  border-radius: 4px;
+  border: 1px solid var(--grey400);
+
+  &.active {
+    background-color: var(--teal100);
+    border: 1px solid var(--teal500);
+
+    .filterButtonIcon {
+      color: var(--teal600);
+    }
+  }
+
+  &:hover {
+    background-color: var(--white, #FFF);
```

Comment:
Just wanted to REALLY double-check on this 😂 It doesn't look like any other hover effect in Finances or the older app styles that I know of.

Reply:
I think it's entirely different for sure! But I followed the buttons spec here https://www.figma.com/file/96NZ02oYe3jpet1roUp0s0/Mockups?type=design&node-id=5385-32459&mode=design&t=b7fbFPa9zO43Y15x-0